.Lib9c.Tests/Model/State/WeeklyArenaStateTest.cs

```diff
@@ -110,5 +110,111 @@ public void ArenaInfoGetRewardCount(int minScore, int maxScore, int expected)
             var info = new ArenaInfo(serialized);
             Assert.Equal(expected, info.GetRewardCount());
         }
+
+        [Theory]
+        [InlineData(100, 1, 100)]
+        [InlineData(100, 51, 50)]
+        [InlineData(10, 1, 50)]
+        [InlineData(10, 1, 1)]
+        [InlineData(10, 6, 50)]
+        [InlineData(10, 6, 1)]
+        public void GetArenaInfos_By_FirstRank_And_Count(int infoCount, int firstRank, int count)
+        {
+            var expected = Math.Min(infoCount - firstRank + 1, count);
```

Comment:
I guess it would be better if specifying expected value directly (instead of calculation) like other fixture(e.g. `infoCount`) for readability of this test case.